### PR TITLE
API docs for copy_chunk

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Add an API doc for the copy_chunk API call

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: api/copy_chunk_experimental.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## copy_chunk() <tag type="community">Community</tag>
+TimescaleDB allows you to copy existing chunks to a new location within a
+multi-node environment. This allows each data node to work both as a primary for
+some chunks and backup for others. If a data node fails, its chunks already
+exist on other nodes that can take over the responsibility of serving them.
+
+### Required arguments
+
+|Name|Type|Description|
+|-|-|-|
```
Comment (nikkhils) [inline api/copy_chunk_experimental.md:10]: Argument data types
```
chunk regclass, 
source_node name, 
destination_node name
```

Comment (nikkhils) [reply api/copy_chunk_experimental.md:10]: There's no support for tablespaces YET while copying chunks across the data nodes.

File: api/move_chunk.md [hunk 2]
```diff
@@ -1,34 +1,33 @@
-## move_chunk() <tag type="community">Community</tag> 
+## move_chunk() <tag type="community">Community</tag>
```
Comment (nikkhils) [inline api/move_chunk.md:1]: This documentation applies to the existing move_chunk functionality and NOT the experimental one. So this should not be changed at all.

File: api/move_chunk_experimental.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+levels so that the chunk and hypertable are able to be read for most of the
+process. This comes at a cost of slightly higher disk usage during the
+operation. For a more detailed discussion of this capability, see the [Data
+Tiering][using-data-tiering] documentation.
+
+### Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`chunk`|REGCLASS|Name of chunk to be moved|
```
Comment (nikkhils) [inline api/move_chunk_experimental.md:21]: Argument data types
```
chunk regclass, 
source_node name, 
destination_node name
```

File: api/move_chunk_experimental.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+|Name|Type|Description|
+|-|-|-|
+|`reorder_index`|REGCLASS|The name of the index (on either the hypertable or chunk) to order by|
+|`verbose`|BOOLEAN|Setting to true will display messages about the progress of the move_chunk command. Defaults to false.|
+
+
+### Sample usage
+
+``` sql
+SELECT move_chunk(
```
Comment (nikkhils) [inline api/move_chunk_experimental.md:36]: The sample call should be same as `copy_chunk`

**Comment by ryanbooz** [comment]
@nikkhils - if you could give `copy_chunk` and `move_chunk` (experimental) one more look, I'd greatly appreciate it!

**Review by nikkhils — APPROVED** [review]
LGTM  from my point of view @ryanbooz.

I also asked @erimatnor to take a look at these doc changes.

**Pull request merged**